docs/manual/docs/cmd/teams/report/report-useractivityuserdetail.md

```diff
@@ -15,6 +15,8 @@ Option|Description
 `--help`|output usage information
 `-p, --period <period>`|The length of time over which the report is aggregated. Supported values `D7, D30, D90, D180`. Specify the `period` or `date`, but not both.
 `-d, --date [date]`|The date for which you would like to view the users who performed any activity. Supported date format is `YYYY-MM-DD`. Specify the `date` or `period`, but not both.
+`-f, --outputFile [outputFile]`|Path to the file where the upgrade report should be stored in.
+`-o, --output [output]`|Output type. `csv|json|text`. Default `csv`
```

Comment:
The `output` option is there for every command and by default is `text`. I would suggest you keep it the same as the rest of the commands where it defaults to `text` and the options are in the following order `text,json,csv`. See how Waldek did it here: https://github.com/pnp/office365-cli/pull/1037/files

Reply:
I have fixed this in the latest commit.